src/runtime_src/core/common/api/hw_queue.cpp

```diff
@@ -317,7 +317,7 @@ class qds_device : public hw_queue_impl
   std::cv_status
   wait(size_t timeout_ms) override
   {
-    return m_device->wait_command(m_qhdl, XRT_NULL_BO, static_cast<int>(timeout_ms))
+    return m_device->wait_command(m_qhdl, XRT_INVALID_BUFFER_HANDLE, static_cast<int>(timeout_ms))
```

Comment:
Hmm, would `reinterpret_cast `work here instead, being that `size_t` is signed.
```suggestion
    return m_device->wait_command(m_qhdl, XRT_INVALID_BUFFER_HANDLE, reinterpret_cast<int>(timeout_ms))
```

Reply:
Oh no, static_cast is what you want here, it declares the intent of narrowing the type.  Don't use reinterrpret_cast unless you really have to.